src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
Indeed `spark` is correct, even for the new format: https://github.com/quasar-analytics/quasar/issues/1900#issuecomment-285041338